<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentManagement</class>
 <widget class="QWidget" name="studentManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>studentManagement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>学生数据</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBoxSort">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>排序</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>排序字段</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboFields"/>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBtnAscend">
              <property name="text">
               <string>升序</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBtnDescend">
              <property name="text">
               <string>降序</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxFilter">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>数据过滤</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QRadioButton" name="radioBtnMan">
              <property name="text">
               <string>男</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBtnWoman">
              <property name="text">
               <string>女</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBtnBoth">
              <property name="text">
               <string>全显示</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>搜索学生号</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true">alternate-background-color: rgb(211, 249, 248);</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>当前记录</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string/>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>学号</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="dbSpinEmpNo"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>姓名</string>
              </property>
              <property name="buddy">
               <cstring>dbEditName</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="dbEditName"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>性别</string>
              </property>
              <property name="buddy">
               <cstring>dbComboSex</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="dbComboSex">
              <item>
               <property name="text">
                <string>男</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>女</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>出生日期</string>
              </property>
              <property name="buddy">
               <cstring>dbEditBirth</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="dbEditBirth">
              <property name="displayFormat">
               <string>yyyy-MM-dd</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
              <property name="date">
               <date>
                <year>2017</year>
                <month>2</month>
                <day>20</day>
               </date>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>省份</string>
              </property>
              <property name="buddy">
               <cstring>dbComboProvince</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="dbComboProvince">
              <property name="editable">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>北京</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>吉林</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>黑龙江</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>内蒙古</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>辽宁</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>新疆</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>西藏</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>甘肃</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>宁夏</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>浙江</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>台湾</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>海南</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>澳门</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>香港</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>四川</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>云南</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>广西</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>广东</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>福建</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>上海</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>天津</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>重庆</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>安徽</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>河北</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>河南</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>湖北</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>湖南</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>山西</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>山东</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>陕西</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>江苏</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>江西</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>密码</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="dbpswd"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>照片</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="dbLabPhoto">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>350</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>照片在此显示</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>功能栏</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="actRecInsert">
            <property name="text">
             <string>插入</string>
            </property>
            <property name="icon">
             <iconset resource="../../rec.qrc">
              <normaloff>:/按钮图标/images/left-circle.png</normaloff>:/按钮图标/images/left-circle.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="actRevert">
            <property name="text">
             <string>取消</string>
            </property>
            <property name="icon">
             <iconset resource="../../rec.qrc">
              <normaloff>:/按钮图标/images/close-circle.png</normaloff>:/按钮图标/images/close-circle.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="actPhotoClear">
            <property name="text">
             <string>清除照片</string>
            </property>
            <property name="icon">
             <iconset resource="../../rec.qrc">
              <normaloff>:/按钮图标/images/sync.png</normaloff>:/按钮图标/images/sync.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="actPhoto">
            <property name="text">
             <string>设置照片</string>
            </property>
            <property name="icon">
             <iconset resource="../../rec.qrc">
              <normaloff>:/按钮图标/images/image.png</normaloff>:/按钮图标/images/image.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="actSubmit">
            <property name="text">
             <string>保存</string>
            </property>
            <property name="icon">
             <iconset resource="../../rec.qrc">
              <normaloff>:/按钮图标/images/save.png</normaloff>:/按钮图标/images/save.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="actRecDelete">
            <property name="text">
             <string>删除</string>
            </property>
            <property name="icon">
             <iconset resource="../../rec.qrc">
              <normaloff>:/按钮图标/images/minus-circle.png</normaloff>:/按钮图标/images/minus-circle.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="actRecAppend">
            <property name="text">
             <string>添加</string>
            </property>
            <property name="icon">
             <iconset resource="../../rec.qrc">
              <normaloff>:/按钮图标/images/plus-circle.png</normaloff>:/按钮图标/images/plus-circle.png</iconset>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rec.qrc"/>
 </resources>
 <connections/>
</ui>
